pkg/cmd/edge_applications/publish/publish.go

```diff
@@ -357,11 +355,11 @@ func (cmd *publishCmd) fillUpdateRequestFromConf(client *api.Client, ctx context
 	if err != nil {
 		return 0, fmt.Errorf(msg.ErrorUpdateFunction.Error(), err)
 	}
-	fmt.Fprintf(cmd.f.IOStreams.Out, msg.EdgeApplicationsPublishOutputEdgeFunctionUpdate, response.GetName(), idReq)
+	fmt.Fprintf(cmd.F.IOStreams.Out, msg.EdgeApplicationsPublishOutputEdgeFunctionUpdate, response.GetName(), idReq)
```

Comment:
if exporting "F" is not necessary, you can remove it!

Reply:
Is necessary